### PR TITLE
Refactor some code to use simply format

### PR DESCRIPTION
**Pull request by HuKeping**
Signed-off-by: Hu Keping hukeping@huawei.com

### CODE REVIEW

File: graph/history.go [hunk 1]
```diff
@@ -38,8 +37,7 @@ func (graph *Graph) depth(img *image.Image) (int, error) {
 
 	for parent != nil {
 		count++
-		parent, err = graph.GetParent(parent)
-		if err != nil {
+		if parent, err = graph.GetParent(parent); err != nil {
```
Comment (runcom) [inline graph/history.go]: you could even remove `err error` declaration above https://github.com/docker/docker/pull/16103/files#diff-834986d10d3853e020577ea23e648c83R35 and do `if parent, err :=` here (with colon)

Comment (HuKeping) [inline graph/history.go]: Nop,,maybe it's a bug of Golang, but the compiler of Golang will not take `parent` as an `old` variable if we change these code as 
```
...
if parent, err := graph.GetParent(parent); err != nil {
}
...
```

The compiler will error out
```
graph/history.go:39: parent declared and not used
```

Comment (runcom) [inline graph/history.go]: oh you're right, my bad, didn't noticed parent was declared above

**Comment by runcom** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Pull request merged**